### PR TITLE
Help the user fix the website

### PR DESCRIPTION
**Pull request by atjn**
- Provide more feedback that explains how to improve the score for each check.
- Include a link to the Sustainable Web Design entry for this check.
- Fix a few scores in the format plugin.

The user preference plugin is not updated in this PR because I am working on it in a separate PR, please ignore it for now.

### CODE REVIEW

**Review by AlbertHald — DISMISSED** [review]
LTMG (jeg er yoda)

**Review by AlbertHald — APPROVED** [review]
GLTM

File: tests/e2e/querySelector.test.ts [hunk 1]
```diff
@@ -163,5 +163,5 @@ test("make sure without include it does find both resources", async ({
 		.textContent();
 
 	// Assert that the score is 38
```
Comment (BaBrixx) [inline tests/e2e/querySelector.test.ts:165]:
```suggestion
	// Assert that the score is 50
```

**Review by BaBrixx — CHANGES_REQUESTED** [review]

File: tests/e2e/querySelector.test.ts (hunk 1 above)
Comment (atjn) [reply tests/e2e/querySelector.test.ts:165]: tihi

**Pull request merged**